src/download.js

```diff
@@ -8,26 +8,30 @@ const pipeline = promisify(stream.pipeline);
 
 const { fatal } = require('./utils/logging');
 
-const MB_BYTES = 1025 * 1024;
+const MB_BYTES = 1024 * 1024;
```

Comment:
Just curious, why the bump from 1025 -> 1024 here?

Reply:
A megabyte is 1024 * 1024, idk why it was 1025 before, that was just wrong afaics